pkg/utils/rest/kubecontrol.go

```diff
@@ -65,6 +65,10 @@ func NewKubeControl(mapper meta.RESTMapper, config *rest.Config) *KubeControl {
 	}
 }
 
+// TODO:
+// If the resourceOrKindArg format is resource.group, the schema.ParseResourceArg return value
```

Comment:
this is not quite clear.

it might be

The resourceOrKindArg should be in the format of {resource}.{version}.{group}, or {resource}. If it is {resource}.{group}, it will be mistakenly parsed to {resource: {resource}, version: {group} } which will trigger the reload of restmapper and introduce performance degradation. 
TODO: refactor restmapper to avoid frequent reload.